applications/MappingApplication/mapping_application.h

```diff
@@ -142,6 +146,11 @@ class KratosMappingApplication : public KratosApplication {
 	///@}
 	///@name Protected member Variables
 	///@{
+  
+  // Needed for Volume Mapping
```

Comment:
My recommendation would be to have 'high level' functions that accept lists of Elements or Conditions, depending on the end-user, but work with the underlying geometry of the object (that is, a `Geometry< Node<3> >`) for anything that operates on a single element/condition entity. I don't know if this would be easy or a pain to do in practice, though.